configs/components/xios/xios.yaml

```diff
@@ -43,6 +43,8 @@ choose_version:
         archfile: ESMTOOLS_generic_oasis_intel
       parastationmpi:
         archfile: ESMTOOLS_generic_oasis_intel
+      "*":
+        archfile: ESMTOOLS_generic_oasis_intel
```

Comment:
This was needed since HLRN does not have the `useMPI` variable, and I did not want to add it. @JanStreffing , @sveta-loza , is `ESMTOOLS_generic_oasis_intel` the correct archfile to use for `xios` in HLRN?

Reply:
Sounds right. The default is intel compiler. Only Aleph uses something different so far with cray ftn.